core/async/src/examples/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod simple;
```

Comment:
so these 2 files are actually a part of a single example? (as I considered them separate)

Reply:
Hopefully the renaming you suggested would help?